src/extension.ts

```diff
@@ -31,26 +31,35 @@ export function activate(context: vscode.ExtensionContext) {
 export function deactivate() {
 }
 
-function onOpen(e) {
+function onOpen() {
 
     if (isEnable === true)
-        goToLastLine();
+        goToLastLine(vscode.window.activeTextEditor);
 }
 
 function onChange(e) {
 
     if (isEnable === true && !e.document.isDirty)
-        goToLastLine();
+        goToLastLineForDocument(e.document);
 }
 
-function goToLastLine() {
+function goToLastLineForDocument(changedDoc) {
 
-    const editor = vscode.window.activeTextEditor;    
-    const position = new vscode.Position(vscode.window.activeTextEditor.document.lineCount - 1, 0)
+    vscode.window.visibleTextEditors
+        .filter(te => te.document === changedDoc)
+        .forEach(goToLastLine);    
+}
+
+function goToLastLine(textEditor) {
+    
+    if(!textEditor)
+        return
+
+    const position = new vscode.Position(textEditor.document.lineCount - 1, 0)
 
-    editor.selection =  new vscode.Selection(position,position);
-    editor.revealRange(new vscode.Range(position,position),
-        vscode.TextEditorRevealType.InCenterIfOutsideViewport);
```

Comment:
Is there any reason for changing RevealType?

Reply:
I changed that to fit tools like BareTail a little better. Those tools keep the last line at the bottom of the screen, which gives the user the full screen of text and also keeps the lines from jumping around too much.

Since that isn't part of the issue that this PR address though, I can move that change out of this PR and open an issue where we could discuss it if you'd rather!

Reply:
Thank you.
No need to change it. I have a plan to do some change in extension setting. I think we can make it optional.

Thank you again for your great effort.

Reply:
@PejmanNik Sure thing! If you want to put some of those plans as issues as well, I'd be happy to help out if I have some time!

Reply:
@sperry94 Thank you very much.